docs/source/dsp_chain_in_detail.rst

```diff
@@ -45,39 +47,43 @@ below.
 |      3       | 16.667 kHz | 5                 | 3.333 kHz   | 4              |
 +--------------+------------+-------------------+-------------+----------------+
 
-For a typical sequence of *normalscan* data, the input data to the first filter stage contains 451500 complex
-data samples for each antenna. After the first stage, this has been reduced to 45084 samples per antenna, but if
-multiple frequencies are being selected, there will be one copy of (:math:`num_antennas`, 45084) for each frequency. After
-the second stage, the number of samples is reduced from 45084 to 8992. The next stage reduces it again to 1495 samples.
-Lastly, the final stage reduces the data length to 299 samples. The number of samples in and out of each stage does not
-exactly correspond to the downsampling rate; this will be explained shortly, as a result of the filtering technique.
+For a typical sequence of *normalscan* data, the input data to the first filter stage contains
+451500 complex data samples for each antenna. After the first stage, this has been reduced to 45084
+samples per antenna, but if multiple frequencies are being selected, there will be one copy of
+(:math:`num_antennas`, 45084) for each frequency. After the second stage, the number of samples is
```

Comment:
Fixed here: 49b3fad1df8bb8dfedde8a41e55116a493f9dfa4

Problem is possibly due to me moving the config files outside of the source code, but I fixed it by embedding the latex math images into the HTML, instead of referencing images saved on disk.